### PR TITLE
Migrate fluxible-router to monorepo

### PR DESCRIPTION
**Pull request by mridgway**

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by yahoocla** [comment]
CLA is valid!

File: packages/fluxible-router/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Contributing Code to `fluxible-router`
```
Comment (redonkulus) [inline packages/fluxible-router/CONTRIBUTING.md]: Update all links

File: packages/fluxible-router/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Fluxible Router
```
Comment (redonkulus) [inline packages/fluxible-router/README.md]: Update all links and badges

**Pull request merged**